gitops/argocd/tekton-results/README.md

```diff
@@ -0,0 +1,10 @@
+# Tekton Results
+
+## ***Description*** 
+This will install tekton results on cluster to gather Tekton Pipelinerun results for long term storage.
+
+Installation is based on the [installation instructions](https://github.com/tektoncd/results/blob/main/docs/install.md) from upstream tekton results
```

Comment:
Should we rather point to our fork?

Reply:
I think it'll be cleaner to confirm this PR works as expected and then update what's needed for our fork and the included changes we may need once this is merged.